cache/filesystem_cache_test.go

```diff
@@ -106,7 +100,11 @@ func cacheAddGetHelper(t *testing.T, c Cache, c1 Cache) {
 		}
 		// Verify trw contains the response.
 		if buf.String() != value {
-			t.Fatalf("unexpected response sent to client: %q; expecting %q", trw.b, value)
+			conditionalStr := ""
+			if len(value) > 30 {
+				conditionalStr = "..."
+			}
```

Comment:
It's odd, I guess it's to display the fact that we display only a fraction of data. Is it needed? If you think so, maybe we can mutualise `30` to variable and rename `conditionalStr` to `logSuffx`?

Reply:
it's needed because I added a test with a string of 4MB and if this test fail the logs will contains a string of 4MB ...
I'll do the changes requested